tools/run-smarten.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+set -eu -o pipefail
+
+declare -r SPEC_DIRECTORY=../standard
+declare -r EXTENSION=md
+
+declare -a SPEC_FILES=(
+    "foreword"
+    "introduction"
+    "scope"
+    "normative-references"
+    "terms-and-definitions"
+    "general-description"
+    "conformance"
+    "lexical-structure"
+    "basic-concepts" 
+    "types"
+    "variables"
+    "conversions"
+    "expressions"
+    "statements"
+    "namespaces"
+    "classes"
+    "structs"
+    "arrays"
+    "interfaces"
+    "enums"
+    "delegates"
+    "exceptions"
+    "attributes"
+    "unsafe-code"
+    "grammar"
+    "portability-issues"
+    "standard-library"
+    "documentation-comments"
+    "bibliography"
+    )
+
+# unpack the package to ./smarten
+tar -xvf ../.github/workflows/dependencies/EcmaTC49.Smarten.tar
+
+# On some files, smarten can't "round trip", so creates
```

Comment:
What exactly is meant by "can't round trip" here?

Reply:
I'd have to dig deeper into the smarten source. On (almost) every file, the smarten_CL writes the same destination file as the source (as we want). One these two, it fails to do that, and writes a different file destination.

Those are the largest files in the standard. I'm suspecting it has something to do with the file size, but I haven't figured that out yet.